rq/cli/helpers.py

```diff
@@ -48,12 +49,22 @@ def get_redis_from_config(settings, connection_class=Redis):
         sn = Sentinel(instances, socket_timeout=socket_timeout, password=password, db=db)
         return sn.master_for(master_name)
 
+    ssl = settings.get('REDIS_SSL', False)
+    if isinstance(ssl, string_types):
+        if ssl.lower() in ['y', 'yes', 't', 'true']:
+            ssl = True
+        elif ssl.lower() in ['n', 'no', 'f', 'false', '']:
+            ssl = False
+        else:
+            raise ValueError('REDIS_SSL is a boolean and must be \'y\', \'yes\', \'t\', \'true\', \'n\', \'no\', '
```

Comment:
I think the error message could be a bit simpler. We can just say `REDIS_SSL is a boolean and must be "True" or "False"`